src/utils/typeStrings/ast/inaccessible_dynamic_type.ts

```diff
@@ -0,0 +1,11 @@
+import { TypeNode } from 'solc-typed-ast';
```

Comment:
I wonder why `inaccesible_dynamic_type` and `module_type` are needed

Reply:
The idea is to modify the typestring parser as little as possible to minimize the maintenance of it. As I said below it's a temporary solution, eventually the whole typeStrings directory should be removed